azure/references/SkuProfile/id.ftl

```diff
@@ -0,0 +1,167 @@
+[#ftl]
+
+[@addReference
+    type=SKU_PROFILE_REFERENCE_TYPE
+    pluralType="SkuProfiles"
+    properties=[
+            {
+                "Type"  : "Description",
+                "Value" : "Compute SKU model and hardware type."
+            }
+        ]
+    attributes=[
+        {
+            "Names" : "apigateway",
+            "Children" : [
+                {
+                    "Names" : "Name",
+                    "Type" : STRING_TYPE,
+                    "Values" : [ "Developer", "Standard", "Premium", "Basic", "Consumption" ]
+                },
+                {
+                    "Names" : "Capacity",
+                    "Type" : NUMBER_TYPE
+                }
+            ]
+        },
+        {
+            "Names" : [ "storage", "s3", "baselinedata" ],
```

Comment:
An array of names is actually used for backwards compatibility rather than duplication. From memory any array member after the first one will be replaced with the first name after configuration processing

Reply:
updated to s3 only